datalad/distribution/update.py

```diff
@@ -154,13 +183,17 @@ def _update_repo(ds, remote, merge, fetch_all, reobtain_data):
         if reobtain_data:
             # get all annexed files that have data present
             lgr.info('Recording file content availability to re-obtain update files later on')
-            reobtain_data = repo.get_annexed_files(with_content_only=True)
+            reobtain_data = \
+                [opj(ds.path, p)
```

Comment:
why do we need full path here if later it is used in  `ds.get` anyways which would probably strip those off directly or indirectly (by annex)?